tools/Makefile

```diff
@@ -19,12 +17,13 @@ SCTK_GITHASH = 20159b5
 
 # Default features configured for OpenFST; can be overridden in the make command line.
 OPENFST_CONFIGURE ?= --enable-static --enable-shared --enable-far \
-                     --enable-ngram-fsts --enable-lookahead-fsts --with-pic
+                     --enable-ngram-fsts --enable-lookahead-fsts \
+                     --disable-dependency-tracking --with-pic
```

Comment:
Is `--disable-dependency-tracking` really a good idea? You are basically assuming that users won't modify and develop openfst source code, which is something I've done in the past.

Reply:
When automatically compiling it as part of Kaldi, I think it's a good idea. If you really hack OpenFST in context of Kaldi to the point dependencies change, you'll probably rerun configure anyway many times.

Reply:
I think the --disable-dependency-tracking  flag should be removed. I don't want to optimize for the time of machine. I want to optimize for time of people who are trying to track down an issue.

Reply:
Done.